### ISSUE TITLE
Can we see brokers list as well

### ISSUE THREAD

**Issue by jaydeepjc** [linkedin/Burrow#456]
Can I see brokers list of kakfa as well from Burrows?
If yes, How can It be done. please suggest.

**Comment by thomaslaw** [comment]
You can get cluster details from burrow:

https://github.com/linkedin/Burrow/wiki/http-request-kafka-cluster-detail

Is this what you want?

**Comment by jaydeepjc** [comment]
Yes, However not only the brokers lists but the how many brokers are actually online .

**Comment by toddpalino** [comment]
By definition, the brokers that are returned are the ones that are online, as it's built from cluster metadata. If there is a broker that is part of the cluster, but is down, it will not show up.
